packages/IMAPClient-UI.package/ICLoginDialog.class/instance/buildWith..st

```diff
@@ -85,10 +85,19 @@ buildWith: builder
 
 	window := builder build: windowSpec.
 	
+	window on: #keyStroke send: #keyEvent: to: self.
+	self loginMorph: window.
+	
+	self inputCollection: OrderedCollection new.
+	
+	
+	window submorphsDo: [ :m |
+	"add listener to text fields"	
+		(m class == PluggableTextMorphPlus)
+			ifTrue: [self inputCollection add: m.
+					m textMorph on: #keyDown send: #keyEvent: to: self].
```

Comment:
When you use [] you always leave space at the front like this: [ something ...,
but you never leave space at the end, like: [ something] :D.